apps/agreement/contracts/Agreement.sol

```diff
@@ -530,8 +518,9 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
         bool finishedSubmittingEvidence = challenge.submitterFinishedEvidence && challenge.challengerFinishedEvidence;
         require(finishedSubmittingEvidence && !challenge.evidencePeriodClosed, ERROR_CANNOT_CLOSE_EVIDENCE_PERIOD);
 
+        challenge.evidencePeriodClosed = true;
```

Comment:
Just sanity checking, we can't rely on the arbitrator to revert if it's already closed for them?

Otherwise this state can get out of sync if nobody calls `closeEvidencePeriod()`.

Reply:
Totally! Fixed now :)